.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
The `.idea/` tree can be removed, these files are specific to an IDE that not everyone uses.